deploy.yaml

```diff
@@ -198,6 +197,20 @@
       - mosca
       - lwm2m
 
+    - role: influxdb-storer
```

Comment:
Is it possible to disable history / persister?

Reply:
Maybe it could have a tag `disable persister/history`

Reply:
Hi @mprevide, we can use "--skip-tags="influxdb" . This command ignores everything that has the influxdb tag.

Reply:
But what if I want to take away persister and history?

Reply:
I thought that for now MongoDB and the services that come with it were not optional.

Reply:
So, I understood that it would be possible to disable only persister and history, keeping MongoDB. The only place that would have any problem would be in the GUI in the details.

Reply:
@rascaraficci 
What do you think it's better? Maybe for now not having that option then?

Reply:
Maybe for now, something like: **ansible-playbook -K -k -i inventories/example_local/ deploy.yaml --skip-tags="persister,history"**?

Reply:
That was changed.